swagger.yml

```diff
@@ -125,6 +234,29 @@ definitions:
             type: string
           reviews:
             type: string
+  Review:
+    type: object
+    required:
+      - message
```

Comment:
`id`, `last_updated` and `book_id` are all required fields currently missing from your list